### PR TITLE
docs: add docs on cookieless

### PR DESCRIPTION
**Pull request by johanbook**
First draft on docs for cookieless. Note that it **will not be visible in sidebar**, only by navigating directly to the URL

### CODE REVIEW

File: docs/docs/client-side/cookieless.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Cookieless analytics
```
Comment (LiKang6688) [inline docs/docs/client-side/cookieless.md:1]: shall we call it Cookieless analytics or Cookieless tracking?

**Comment by LiKang6688** [comment]
I think that it would be good to talk about What is cookieless analytics, the benefits and disadvantages of cookieless analytics.
https://matomo.org/cookie-consent-banners/ is a good reference.

**Review by LiKang6688 — APPROVED** [review]

File: docs/docs/client-side/cookieless.md (hunk 1 above)
Comment (johanbook) [reply docs/docs/client-side/cookieless.md:1]: It should be Cookieless Account Analytics

**Comment by johanbook** [comment]
> I think that it would be good to talk about What is cookieless analytics, the benefits and disadvantages of cookieless analytics. https://matomo.org/cookie-consent-banners/ is a good reference.

Good point, will give it a go

**Pull request merged**